src/main/java/com/binance/dex/api/client/TransactionConverter.java

```diff
@@ -172,13 +178,14 @@ public Transaction convert(byte[] bytes) {
         }
     }
 
+
```

Comment:
These several changes could be auto formatting. I do not notice this when committing.